src/main/java/org/fogbowcloud/saps/engine/core/catalog/CatalogConstants.java

```diff
@@ -0,0 +1,5 @@
+package org.fogbowcloud.saps.engine.core.catalog;
+
+public class CatalogConstants {
```

Comment:
As I described in the comment on the Archiver code, I think we don't need this constante

Reply:
I removed the constant related to the catalog and corrected the code so as not to use a limiter because in all cases it was not useful, believing it was an early optimization.